src/web_app/js/sdputils.js

```diff
@@ -34,6 +35,20 @@ function iceCandidateType(candidateStr) {
   return candidateStr.split(' ')[7];
 }
 
+// Turns the local type preference into a human-readable string.
+// Note that this mapping is browser-specific.
+function formatTypePreference(pref) {
```

Comment:
oh... it's currently not called in Firefox at all since that doesn't support the stats code (yet). Anyway,   it might in the future.